.env.template

```
@@ -27,3 +27,10 @@ DEFAULT_ADMIN_PASSWORD=
 
 # CORS Config
 WEBAPP_HOST=http://localhost:5173
+
+# Port to use in the express server
+API_PORT=
+
+# Rate limiter config
+RATE_WINDOW=
+RATE_REQ_NUMBER=
```

Deixar claro que é o máximo de requests.
```suggestion
RATE_MAX_REQS=
```